content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -85,10 +85,10 @@ spec:
 
 The general workflow of a device plugin includes the following steps:
 
-* Initialization. During this phase, the device plugin performs vendor specific
+1. Initialization. During this phase, the device plugin performs vendor specific
   initialization and setup to make sure the devices are in a ready state.
 
-* The plugin starts a gRPC service, with a Unix socket under host path
+2. The plugin starts a gRPC service, with a Unix socket under host path
```

Comment:
Even for an ordered list, we don't explicit use integers other than `1`. The Markdown parser will figure out the correct number to use.
The benefit of always using `1` is that we can always insert/remove the items without having to renumber the whole list again.

Reply:
updated it, Thanks.